cli/utils.go

```diff
@@ -43,6 +43,19 @@ func getFileContents(file string) (string, error) {
 	return string(bytes), nil
 }
 
+// From the given path to the storage determine
+// if it's a kubernetes ephemeral storage
+func IsEphemeralStroage(path string) bool {
+	splitSourceSlice := strings.Split(path, "/")
+	if len(splitSourceSlice) > 1 {
+		storageType := splitSourceSlice[len(splitSourceSlice)-2]
+		if storageType == "kubernetes.io~empty-dir" {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
I wish I had a better way here. :( 

In future, if k8s decides to change that path, kata might just end up provisioning the storage over 9pfs just like it does right now. 

I am trying to see if we can come up with some sort of standard for ephemeral storage that everyone can comply with (k8s, runtimes etc). Until then, I don't see any other way.

Reply:
/cc @saad-ali

Reply:
@jodh-intel @WeiZhang555 @sboeuf I have created a topic in k8s community on the stability of ephemeral volumes they provision, https://groups.google.com/forum/#!topic/kubernetes-dev/4JtfGSSUO_A

Feel free to join the discussion.

Reply:
Nice, thx @harche.
In the meantime, just make sure our code has enough comment to explain this is not a perfect/stable solution.

Reply:
@sboeuf sure. 

Meanwhile, Looking at Michelle Au's (Google) [comment](https://kubernetes.slack.com/messages/C09QZFCE5/) and Tim Hockin's (Google) [comment](https://groups.google.com/d/msg/kubernetes-dev/4JtfGSSUO_A/q3PfCLJsAQAJ), the general tone there resonates some of the concerns raised here. 

However, Michelle mentioned in her reply on slack,
`to answer your question, emprydir path on the kubelet should be stable` 

Quoting's Tim's reply on mailing list,
`I would probably NAK any change to the volume implementations that changed the path, but I don't think it is guaranteed.`

Although for the longer term, the we need to come up with a solution that doesn't depend on these assurances (or lack of them). There are a couple of approaches, 

1. As Michelle suggested, `we need to find a way to generically invoke volume plugins in the runtime`

2. Or In my opinion, we should decouple storage drivers from kata. Storage drivers should be these pluggable entities which anyone can write for the storage they are dealing with. 

@egernst @WeiZhang555 @sboeuf @bergwolf @jodh-intel 
Let me know your thoughts on how should we proceed in short term as well as in long term.

Reply:
Since this is still a reasonable improvement especially on security aspect, I'm fine to merge it as a short term enhancement.
But it's better to find a solid method doing this later.
LGTM for now to merge this.